docs/source/usrman/envvars.rst

```diff
@@ -0,0 +1,61 @@
+.. _environment:
+
+Environment variables
+=====================
+
+Below we list the environment variables added since mpi4py v3.1.0. The environment
+variables overwrite their corresponding Python attributes at import time of the ``MPI``
+module.
+
+.. envvar:: MPI4PY_RC_INITIALIZE
+
+  Default: ``True``
+
+  Whether to automatically initialize MPI at import time of the ``MPI`` module.
+  Corresponds to ``mpi4py.rc.initialize``.
+
+.. envvar:: MPI4PY_RC_FINALIZE
+
+  Default: ``None``
+
+  Whether to automatically finalize MPI at the exit time. Corresponds to
+  ``mpi4py.rc.finalize``.
+
+.. envvar:: MPI4PY_RC_THREADS
+
+  Default: ``True``
+
+  Whether to initialize MPI with thread support. Corresponds to ``mpi4py.rc.threads``.
+
+.. envvar:: MPI4PY_RC_THREAD_LEVEL
+
+  Default: ``'multiple'``
```

Comment:
If you use double quotes in the description, you should also use double quotes for the default value.

Reply:
Good catch. I fixed.